projects/storefrontapp-e2e-cypress/cypress/integration/regression/wish-list.e2e-spec.ts

```diff
@@ -9,11 +11,13 @@ describe('Wish list', () => {
   });
 
   beforeEach(() => {
-    cy.restoreLocalStorage();
+    clearAllStorage();
```

Comment:
Since we only have one test case, I don't think we need this (nor the afterEach block)

Reply:
This call along with afterEach block are executed before/after each spec so I think they are needed

Reply:
👍🏼